qbft/round_change.go

```diff
@@ -271,6 +272,28 @@ func validRoundChangeForData(
 
 		return nil
 	}
+
+	return nil
+}
+
+func validRoundChangeForDataWithVerification(
```

Comment:
I don't want to nit pick on the naming, but `WithVerification` and `IgnoreSignature` don't sound related to each other when and fact the functionality should be identical with one difference. 
I would go with either "WithVerification" and "NoVerification"
or "IgnoreSignature" and "WithSignature"/"VerifySignature"
but leaving it like now is a bit confusing imo

Reply:
Definitely. Forgot about that. Thanks!